src/utility/HCICordioTransport.cpp

```diff
@@ -103,6 +103,9 @@ extern "C" void wsf_mbed_ble_signal_event(void)
 }
 #endif //CORDIO_ZERO_COPY_HCI
 
+#undef WSF_MS_PER_TICK
+#define WSF_MS_PER_TICK 10
```

Comment:
@facchinm I'm not sure if this should be changed in the core instead?

`WSF_MS_PER_TICK ` defaults to 10ms: https://github.com/arduino/ArduinoCore-nRF528x-mbedos/blob/beac74ca3cd9d07363f66cf9cda6b143e4385cd2/cores/arduino/mbed/features/FEATURE_BLE/targets/TARGET_CORDIO/stack/wsf/include/wsf_timer.h#L40 - but in the Nano 33 BLE variant it's set to 1 ms for some reason (https://github.com/arduino/ArduinoCore-nRF528x-mbedos/blob/a1816cd20de39ffc03a9cfd6b378e7b63b210a89/variants/ARDUINO_NANO33BLE/mbed_config.h#L336).

Reply:
Alternatively I think we can rename it in the file, as it doesn't seem to be compiled in mbed?

Reply:
Renaming it here should work just fine.

Reply:
@facchinm great, just pushed. This is ready for review now.

Later on we could be a bit smarter and change the poll interval dynamically based on the connection, advertising or scan intervals. However, let's see how feedback from customers goes.